### DISCUSSION TITLE
Grid doesn't work on playground

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/panda#1810] [Q&A] [answered]
I am trying to implement this: https://play.panda-css.com/pEu6MWzGgU

**Comment by astahmer** [comment]
you can do it like this https://play.panda-css.com/pEu6MWzGgU/4FoYqbatPl

**Comment by juliomuhlbauer** [comment]
Doesn't the array syntax works in grid?

Does not show either with object:

![image](https://github.com/chakra-ui/panda/assets/53458125/018f850a-7540-4d70-bfca-1ad735df8eca)

**Reply by anubra266** [reply]
@juliomuhlbauer your width of `100%` can't work, that's css not panda. No way for css to know what width it's spanning, since none of the parents have a defined width.
https://play.panda-css.com/4FoYqbatPl/4FuPKMzg09

And as for the array syntax, it's a limitation for just the `columns` property in the `grid` pattern, because of the definition of the `grid` pattern in the base preset.

**Reply by juliomuhlbauer** [reply]
Thanks! I think we should accept the array sintax on the columns prop on the grid pattern by default. Because it would align with all the design system.

**Reply by anubra266** [reply, marked as answer]
@juliomuhlbauer https://github.com/chakra-ui/panda/pull/1816
